src/pkg/cli/subscribe.go

```diff
@@ -23,7 +23,7 @@ func Subscribe(ctx context.Context, client client.Client, services []string) (<-
 		serviceStatus[service] = string(ServiceUnknown)
 	}
 
-	statusChan := make(chan *map[string]string, len(services))
+	statusChan := make(chan map[string]string, len(services))
```

Comment:
I'd move this map to the caller and simply return statuses (or `ServiceInfo`) here.